+----------------------------------------------------------------------------
+
+debian/patches/* and lzma.1 are released under the terms of the GNU
+General Public License.
+
+  This program is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License with
+  the Debian GNU/Linux distribution in file /usr/share/common-licenses/GPL;
+  if not, write to the Free Software Foundation, Inc., 51 Franklin St,
+  Fifth Floor, Boston, MA 02110-1301, USA.
+
+ On Debian systems, the complete text of the GNU General Public
+ License, version 2, can be found in /usr/share/common-licenses/GPL-2.
+
+----------------------------------------------------------------------------
--- /dev/null
+++ lzma.1
@@ -0,0 +1,63 @@
+.TH LZMA 1 "May  5 2006" "Mohammed Adnene Trojette"
+.SH NAME
+LZMA - a compression algorithm, based on Lempel Ziv's compression method.
+.SH SYNOPSIS
+.B lzma
+.BR <e|d>
+.BR inputFile
+.BR outputFile
+.BR [switches...]
+.SH DESCRIPTION
+LZMA provides high compression ratio and very fast decompression, so it is very suitable for embedded applications. For example, it can be used for ROM (firmware) compression.
+.TP
+lzma is the default and general compression method of 7z format in 7-Zip program.
+.PP
+.SH FUNCTION LETTERS
+.TP
+.B e
+Encode file
+.TP
+.B d
+Decode file
+.TP
+.B b
+Benchmark
+.SH SWITCHES
+.TP
+.B -a{N}
+Set compression mode - [0, 1], default: 1 (max)
+.TP
+.B -d{N}
+Set dictionary - [0,30], default: 23 (8MB)
+.TP
+.B -fb{N}
+Set number of fast bytes - [5, 273], default: 128
+.TP
+.B -mc{N}
+Set number of cycles for match finder
+.TP
+.B -lc{N}
+Set number of literal context bits - [0, 8], default: 3
+.TP
+.B -lp{N}
+Set number of literal pos bits - [0, 4], default: 0
+.TP
+.B -pb{N}
+Set number of pos bits - [0, 4], default: 2
+.TP
+.B -mf{MF_ID}
+Set Match Finder: [bt2, bt3, bt4, hc4], default: bt4
+.TP
+.B -eos
+Write End Of Stream marker
+.TP
+.B -si
+Read data from stdin
+.TP
+.B -so
+Write data to stdout
+.SH "SEE ALSO"
+.BR 7z(1), 7za(1)
+.SH AUTHOR
+.TP
+Written for Debian by Mohammed Adnene Trojette.
